api/lib/reducers/polkadotV0-reducers.js

```diff
@@ -174,8 +174,8 @@ function delegationReducer(network, delegation, validator, active) {
     validator,
     amount: delegation.value
       ? BigNumber(delegation.value)
-          .times(network.coinLookup[0].chainToViewConversionFactor)
-          .toFixed(6)
+        .times(network.coinLookup[0].chainToViewConversionFactor)
+        .toFixed(6)
       : 0,
```

Comment:
Why this API changes? Linting is not being consistent?

Reply:
yep just linting inconsistencies